package.json

```diff
@@ -49,6 +49,7 @@
     "@babel/code-frame": "^7.0.0",
     "ajv": "^6.10.0",
     "chalk": "^4.0.0",
+    "ci-job-number": "^1.2.2",
```

Comment:
why we need this package? there is only few lines of code
`process.env.CI_JOB_NUMBER ? parseInt(process.env.CI_JOB_NUMBER, 10) : 0`

Reply:
ah, as I can see there is few different ENV checks https://github.com/ai/ci-job-number/blob/master/index.js#L17

Will it be better to have explicit CI argument, instead of overriding CI_JOB_NAME to get eslint applied to every run?

Reply:
Sure, I can rename `CI_JOB_NUMBER`.

What name do you suggest?

Reply:
‘ESLINT_LINT_ONCE’ ?

Reply:
Anyway, linting based on job index looks quite dangerous

Reply:
What do you think about `DISABLE_CI_JOB_IGNORING`?

We need a environment variable not just for ESLint but for the library, to reuse library in other heavy tasks.

For instance, in PostCSS we are calling spelling check only on the first CI job.